README.md

```diff
@@ -1,158 +1,111 @@
+## Deploying CDI
 
-On detecting a new PVC with the endpoint annotation (and lacking the status annotation), the controller creates the Data Importer pod "golden" namespace. The controller performs clean up operations after the data import process ends.
+### Assumptions
+- A running Kubernetes cluster
+- A storage class and provisioner.
+- An HTTP or S3 file server hosting VM images
+- A namespace acting as the image registry (`default` is fine for tire kicking)
 
-**Data Import Pod:** Short-lived pod in "golden" namespace. The pod is created by the controller and consumes the secret (if any) and the endpoint annotated in the PVC. It copies the object referenced by the PVC's endpoint annotation to the destination directory used by the storage class/provider. In all cases the target file **name** is _disk.img_.
+### Either clone this repo or download the necessary manifests directly:
 
-**Dynamic Provisioner:** Existing storage provisoner(s) which create the Golden Persistent Volume that reference an empty cluster storage volume. Creation begins automatically when the Golden PVC is created by an admin.
+`$ git clone https://github.com/kubevirt/containerized-data-importer.git`
 
-**Endpoint Secret:** Long-lived secret in "golden" namespace that is defined and created by the admin. The Secret must contain the access key id and secret key required to make requests from the object store. The Secret is mounted by the Data Import pod.
+*Or*
 
-**"Golden" Namespace:** Restricted/private Namespace for Golden PVCs and endpoint Secrets. Also the namespace where the CDI Controller and CDI Importer pods run.
+```shell
+$ mkdir cdi-manifests && cd cdi-manifests
+$ wget https://raw.githubusercontent.com/copejon/containerized-data-importer/kubevirt-centric-readme/manifests/example/golden-pvc.yaml
+$ wget https://raw.githubusercontent.com/copejon/containerized-data-importer/kubevirt-centric-readme/manifests/example/endpoint-secret.yaml
+$ wget https://raw.githubusercontent.com/copejon/containerized-data-importer/kubevirt-centric-readme/manifests/controller/controller/cdi-controller-deployment.yaml
+```
 
-**Golden PV:** Long-lived Persistent Volume created by the Dynamic Provisioner and written to by the Data Import Pod.  References the Golden Image volume in storage.
+### Run the CDI Controller
 
-**Golden PVC:** Long-lived Persistent Volume Claim manually created by an admin in the "golden" namespace. Linked to the Dynamic Provisioner via a reference to the storage class and automatically bound to a dynamically created Golden PV. The "default" provisioner and storage class is used in the example; however, the importer pod supports any dynamic provisioner which supports mountable volumes.
+Deploying the CDI controller is straight forward.  Create the controller in the namespace where VM images are to be stored.  Here, `default` is used.  In a production setup, a namespace that is inaccessible to regular users should be used instead.
```

Comment:
nit: I'd change _Here, `default` is used.  In a production setup_ to _Here, `default` is used. but in a production setup..._.